packages/mdc-slider/component.ts

```diff
@@ -119,7 +119,16 @@ export class MDCSlider extends MDCComponent<MDCSliderFoundation> {
       setTrackStyleProperty: (propertyName, value) => this.track_.style.setProperty(propertyName, value),
       setMarkerValue: (value) => this.pinValueMarker_.innerText = value.toLocaleString(),
       setTrackMarkers: (step, max, min) => {
-        this.trackMarkerContainer_.style.setProperty('background', 'linear-gradient(to right, currentColor 2px, transparent 0) 0 center / calc((100% - 2px) /  (('+ max.toLocaleString() + ' - ' + min.toLocaleString() + ') / ' + step.toLocaleString() + ')) 100% repeat-x');
+        const maxString = max.toLocaleString();
+        const minString = max.toLocaleString();
+        const stepString = max.toLocaleString();
+        // keep calculation in css for better rounding/subpixel behavior
+        const markerAmount = '((' + maxString + ' - ' + minString + ') / ' + stepString + ')';
```

Comment:
Can you convert these to using backticks? It might read better.

Reply:
Oh definitely, I would rather. I just hadn't come across any template literals in the codebase yet, so I wasn't sure if it was your style.